File: crates/fresh-editor/tests/fixtures/syntax_highlighting/hello.r
# R syntax highlighting test
greet <- function(name) {
  paste("Hello,", name, "!")
}

message <- greet("World")
print(message)

x <- c(1, 2, 3, 4, 5)
mean_x <- mean(x)
if (mean_x > 3) {
  cat("Above threshold\n")
}

File: crates/fresh-editor/tests/fixtures/syntax_highlighting/hello.zig
// Zig syntax highlighting test
const std = @import("std");

fn greet(name: []const u8) void {
    std.debug.print("Hello, {s}!\n", .{name});
}

pub fn main() !void {
    const allocator = std.heap.page_allocator;
    var list = std.ArrayList(u8).init(allocator);
    defer list.deinit();

    greet("World");
    const x: i32 = 42;
    _ = x;
}
